compiler-rt/include/profile/InstrProfData.inc

```diff
@@ -742,9 +744,10 @@ serializeValueProfDataFrom(ValueProfRecordClosure *Closure,
  */
 #define VARIANT_MASKS_ALL 0xffffffff00000000ULL
 #define GET_VERSION(V) ((V) & ~VARIANT_MASKS_ALL)
-#define VARIANT_MASK_IR_PROF (0x1ULL << 56)
-#define VARIANT_MASK_CSIR_PROF (0x1ULL << 57)
-#define VARIANT_MASK_INSTR_ENTRY (0x1ULL << 58)
+#define VARIANT_MASK_IR_PROF (0x1ULL << 55)
+#define VARIANT_MASK_CSIR_PROF (0x1ULL << 56)
+#define VARIANT_MASK_INSTR_ENTRY (0x1ULL << 57)
```

Comment:
To maintain backwards compatibility these bits cannot be changed. I guess we need to expand the mask to support more bits.

Reply:
Are bits 33-55 in use?

Reply:
I see that you are right (historically it started at `56` and climbed higher), but I cannot find usages of bit `55`

Reply:
Considering that there is no usage of bit `55` and that there is no documentation about this being reserved, I would argue that we should be free to use `55`. If any downstream user is relying on such an undocumented implementation detail, we should be free to break it.